template/.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+on: [push]
+
+jobs:
+  static-analysis-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12.2'
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** Limit the version specification to `major.minor` only

There really should not be any breaking changes between `patch` releases of Python. Given that, specifying the version down to the `patch` level means we are likely to need to update it more often than is really necessary. I'd instead change this to just `3.12`.‏

Reply:
updated in https://github.com/ackama/django-template/commit/c31cf55160c355420fa8a874e1795922c1d1ccd6